### PR TITLE
Project analyses

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Refactored Project Analyses pages to use Ant Design.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

**Comment by joshsadam** [comment]
On your new ant design tables, can you make the checkbox column fixed left so that if someone is scrolling horizontally and they see something they want it is easier to select?

**Review by joshsadam — CHANGES_REQUESTED** [review]
This is really great, small things

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java [hunk 1]
```diff
@@ -170,53 +170,13 @@ public String getSynchronizeProjectPage() {
 	 * @param model     model for view variables
 	 * @return name of the analysis view page
 	 */
-	@RequestMapping("/projects/{projectId}/analyses")
+	@RequestMapping("/projects/{projectId}/analyses/**")
 	public String getProjectAnalysisList(@PathVariable Long projectId, Principal principal, Model model) {
 		Project project = projectService.read(projectId);
 		projectControllerUtils.getProjectTemplateDetails(model, principal, project);
 		model.addAttribute("project", project);
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java]: Removed in 5c1786965e98a61df9cf733468a44f7c2c6c5976. It happens :)

File: src/main/webapp/pages/projects/project_analyses.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    xmlns:th="http://www.thymeleaf.org"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    data-layout-decorate="~{projects/_base}"
+>
+<head>
+    <title th:text="${project.label} + ' - ' + #{project.nav.analysis}"></title>
+    <link rel="stylesheet" th:href="@{/dist/css/project-analyses.bundle.css}" />
+</head>
+<body>
+<div layout:fragment="main" id="root"></div>
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/pages/projects/project_analyses.html]: Added in 5c1786965e98a61df9cf733468a44f7c2c6c5976

**Comment by deepsidhu85** [comment]
> On your new ant design tables, can you make the checkbox column fixed left so that if someone is scrolling horizontally and they see something they want it is easier to select?

Added fixed checkbox column to the single sample analysis outputs pages in 5c1786965e98a61df9cf733468a44f7c2c6c5976

**Review by joshsadam — APPROVED** [review]
Thanks dude.

**Pull request merged**